packages/experimental/RadioGroup/src/Radio/Radio.tsx

```diff
@@ -51,8 +47,8 @@ export const Radio = compose<RadioType>({
       );
 
       return (
-        <Slots.root {...mergedProps}>
-          <Slots.button>
+        <Slots.root {...mergedProps} {...(Platform.OS == 'android' && { accessible: !disabled, focusable: !disabled })}>
+          <Slots.button accessible={false} onPress={onPress} disabled={disabled} focusable={false}>
```

Comment:
The button slot has been changed into a pressable to support a separate ripple for Android. Setting accessible and focusable as false is required to prevent any changes to the accessibility flow and restore the old behavior.